chrome-launcher/README.md

```diff
@@ -0,0 +1,71 @@
+# Chrome Launcher
+
+Launch chrome with ease from node.
+
+```
+npm install chrome-launcher
+```
+
+
+## API
+
+#### Launch options
+
+```js
+chromeLauncher.launch({
+  // optional staring url string
+  startingUrl: string;
+  // optional array of (string) flags to pass to chrome
```

Comment:
mention these need `--` or link to http://peter.sh/experiments/chromium-command-line-switches/